### PR TITLE
Bolt/command customization

### PR DESCRIPTION
**Pull request by mbolt35**
## What does this PR change?
* Addition to opencost argument processing to allow for the creation of custom commands. 

## How will this PR impact users?
* No change

### CODE REVIEW

File: pkg/costmodel/clusters/clustermap.go [hunk 1]
```diff
@@ -22,6 +22,25 @@ const (
 	LoadRetryDelay time.Duration = 10 * time.Second
 )
 
+// The following constants are used as keys into the cluster info map data structure
+const (
```
Comment (michaelmdresser) [inline pkg/costmodel/clusters/clustermap.go:26]: :heart:

File: pkg/costmodel/clusters/clustermap.go [hunk 2]
```diff
@@ -391,3 +355,63 @@ func (pcm *PrometheusClusterMap) StopRefresh() {
...
+	if i, ok := info[ClusterInfoIdKey]; ok {
+		id = i
+	} else {
+		return nil, fmt.Errorf("Cluster Info Missing ID")
+	}
+	if n, ok := info[ClusterInfoNameKey]; ok {
+		name = n
+	} else {
+		name = id
+	}
```
Comment (michaelmdresser) [inline pkg/costmodel/clusters/clustermap.go:374]: This case used to be an error, what's the reason for the change?

Comment (mbolt35) [reply pkg/costmodel/clusters/clustermap.go:374]: Poor logic. If we have the id, the name should always inherit the id as the name. Additionally, as we expand the definition of ClusterInfo beyond the scope of just prom, being more precise here is important. Name is not a required cluster info configuration, therefore, we shouldn't error when it's missing.

Comment (michaelmdresser) [reply pkg/costmodel/clusters/clustermap.go:374]: Makes perfect sense to me!

**Review by michaelmdresser — APPROVED** [review]
